app/grandchallenge/algorithms/forms.py

```diff
@@ -555,13 +556,34 @@ class AlgorithmImageUpdateForm(SaveFormInitMixin, ModelForm):
 
     class Meta:
         model = AlgorithmImage
-        fields = ("requires_gpu", "requires_memory_gb")
+        fields = ("requires_gpu", "requires_memory_gb", "comment")
         labels = {"requires_gpu": "GPU Supported"}
         help_texts = {
             "requires_gpu": "If true, inference jobs for this container will be assigned a GPU"
         }
 
 
+class ImageMarkDesiredForm(Form):
```

Comment:
The form is missing the selection of the algorithm image. The algorithm image pk should be sent with the form data and not pulled out of the url and passed through the init method.